aicsimageio/readers/reader.py

```diff
@@ -148,6 +148,29 @@ def get_physical_pixel_size(self, scene: int = 0) -> Tuple[float]:
         """
         return (1.0, 1.0, 1.0)
 
+    def get_channel_names(self, scene: int = 0) -> Optional[List[str]]:
+        """
+        Attempts to use the image's metadata to get the image's channel names.
+
+        Parameters
+        ----------
+        scene: int
+            The index of the scene for which to return channel names.
+
+        Returns
+        -------
+        channels_names: Optional[List[str]]
+            List of strings representing the channel names.
+            If channel dimension not present in file, return None.
+        """
+        # Check for channel in dims
+        if Dimensions.Channel in self.dims:
+            channel_index = self.dims.index(Dimensions.Channel)
+            channel_dim_size = self.dask_data.shape[channel_index]
+            return [str(i) for i in range(channel_dim_size)]
+
+        return None
```

Comment:
Would it be safe to `return ["0"]` here ? On the assumption there is one channel if there is any data at all.